### PR TITLE
Add Firefox 120 profile

### PR DESCRIPTION
**Pull request by jonhdoe3058**

### CODE REVIEW

**Comment by bogdanfinn** [comment]
@jonhdoe3058 thank you for your contribution

File: profiles/contributed_browser_profiles.go [hunk 1]
```diff
@@ -1 +1,144 @@
...
+						tls.PSSWithSHA384,
+						tls.PSSWithSHA512,
+						tls.PKCS1WithSHA256,
+						tls.PKCS1WithSHA384,
+						tls.PKCS1WithSHA512,
+						tls.ECDSAWithSHA1,
+						tls.PKCS1WithSHA1,
+					}},
+					&tls.FakeRecordSizeLimitExtension{0x4001},
+					tls.BoringGREASEECH(),
```
Comment (gaukas) [inline profiles/contributed_browser_profiles.go:86]: I'm pretty sure this was a mistake. Firefox does not use BoringSSL but NSS instead, which uses the same combination of cipher suites but produces different result payload length.

Comment (jonhdoe3058) [reply profiles/contributed_browser_profiles.go:86]: just checked the hash, I get the same one using Firefox_120 as the one I get with my real browser Firefox 123.
http://sprunge.us/Px9nxp
Link above contains result of https://tls.peet.ws/api/all from my actual browser

Comment (gaukas) [reply profiles/contributed_browser_profiles.go:86]: Not sure if the hash counts the **payload length** in, but I don't think so.

Comment (gaukas) [reply profiles/contributed_browser_profiles.go:86]: AgAin, NSS and BoringSSL generates almost identical greased ECH with the only difference being the length of the (greased) Encrypted InnerClientHello (aka the payload).

**Pull request merged**